lib/src/components/breadcrumbs/breadcrumbs.dart

```diff
@@ -0,0 +1,324 @@
+  BreadCrumb createBreadCrumb(BreadCrumb input, int index) {
+    return BreadCrumb(
+      label: input.label,
+      isSelected: _selectedIndex == index,
+      onPressed: () {
+        setState(() {
+          _selectedIndex = index;
+        });
+        input.onPressed.call();
+      },
+      activeIcon: widget.activeIcon,
+    );
+  }
+
+  List<Widget> renderedChildren(List<BreadCrumb> children) {
+    final List<Widget> returnList = [];
+    if (children.length > 3) {
+      returnList.add(createBreadCrumb(children.first, 0));
+
+      final List<Widget> truncatedChildren = [];
+
+      for (final (index, element)
+          in children.sublist(1, children.length - 1).indexed) {
+        truncatedChildren.add(createBreadCrumb(element, index + 1));
+      }
+      returnList
+        ..add(
+          BreadCrumbsTruncated(
+            rounded: widget.rounded,
+            children: truncatedChildren,
+          ),
+        )
+        ..add(createBreadCrumb(children.last, children.length - 1));
+    } else {
+      for (final (index, element) in children.indexed) {
+        returnList.add(createBreadCrumb(element, index));
+      }
+    }
+    return returnList;
+  }
+}
+
+/// Class for untruncated [BreadCrumb].
+class BreadCrumb extends StatelessWidget {
+  ///Constructor for [BreadCrumb]
+  const BreadCrumb({
+    super.key,
+    required this.label,
+    this.icon,
+    this.isSelected = false,
+    required this.onPressed,
+    this.activeIcon,
+  });
+
+  /// [BreadCrumb] label.
+  final String label;
+
+  /// Selected icon.
+  final IconData? icon;
+
+  /// Is [BreadCrumb] selected.
+  final bool isSelected;
+
+  /// Handles press for [BreadCrumb]
+  final VoidCallback onPressed;
+
+  /// Active icon for [BreadCrumb]
+  final IconData? activeIcon;
+
+  @override
+  Widget build(BuildContext context) {
+    final controller = MaterialStatesController();
+    final colors = Zeta.of(context).colors;
+    return Material(
+      color: Colors.transparent,
+      child: InkWell(
+        statesController: controller,
+        onTap: onPressed,
+        enableFeedback: false,
+        splashColor: Colors.transparent,
+        overlayColor: MaterialStateProperty.resolveWith((states) {
+          return Colors.transparent;
+        }),
+        child: Row(
+          children: [
+            if (isSelected)
+              Icon(
+                activeIcon ?? ZetaIcons.star_round,
+                color: getColor(controller.value, colors),
+              ),
+            const SizedBox(
+              width: ZetaSpacing.xs,
+            ),
+            Text(
+              label,
+              style: TextStyle(color: getColor(controller.value, colors)),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  /// Get color of breadcrumb based on state.
+  Color getColor(Set<MaterialState> states, ZetaColors colors) {
+    if (states.contains(MaterialState.hovered)) {
+      return colors.blue.shade100;
```

Comment:
This does not work for 2 reasons -
1. Wrong color - should be shade 60
2. The widget is stateless, so this only gets called when the whole widget changes - this is why it works for isSelected
To fix this, you could make the widget stateful, and add a listener to changes on the controller like this:
```dart

class _BreadCrumbState extends State<BreadCrumb> {
  final controller = MaterialStatesController();
  @override
  void initState() {
    super.initState();
    controller.addListener(() {
      if (context.mounted && mounted) {
        setState(() {});
      }
    });
  }

  @override
  Widget build(BuildContext context) {
...
```

Reply:
Is shade 60 the base version of most colors?
Its hard to deduce from figma

Reply:
yeah shade60 is always the base color. You can also just remove the .shadeX part, and this will always return the base color